BO4E/ENUM/Status.cs

```diff
@@ -60,6 +60,39 @@ public enum Status
         /// <remarks>Z39</remarks>
         ZAEHLERSTAND_ZUM_ENDE_DER_ANGEGEBENEN_ENERGIEMENGE_NICHT_VORHANDEN_DA_MENGENABGRENZUNG,
 
+        /// <summary> KEIN_ZUGANG </summary>
+        /// <remarks>Z74</remarks>
+        KEIN_ZUGANG,
+
+        /// <summary> KOMMUNIKATIONSSTOERUNG </summary>
+        /// <remarks>Z75</remarks>
+        KOMMUNIKATIONSSTOERUNG,
+
+        /// <summary> NETZAUSFALL </summary>
+        /// <remarks>Z76</remarks>
+        NETZAUSFALL,
+
```

Comment:
Fehlt hier nicht noch `Spannungsausfall` ?
```suggestion
         /// <summary> Spannungsausfall </summary>
         /// <remarks>Z77</remarks>
         SPANNUNGSAUSFALL,
```

Reply:
3b67456fcf2be87529080570513fe303c705c65c